greenplumpython/gpapply.py

```diff
@@ -1,7 +1,9 @@
 import inspect
-import psycopg2
 import random
 import string
+import numpy as np
+
+numpyDTypeToGPDBType = {np.dtype('int64'): "int", np.dtype('object'): "text", np.dtype("datetime64[ns]"):"datetime"}
```

Comment:
Any reference for this global variable ?

Reply:
It's useless now, I'll delete it